config/environments/test.rb

```diff
@@ -40,9 +40,6 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
-  # disable auditing so tests run faster
-  Song.auditing_enabled = false
```

Comment:
Did you remove this so the CI will sort of test auditing?

Reply:
when I don't remove it I get
```
/Volumes/workplace/GraceTunes/config/environments/test.rb:40:in `block in <top (required)>': uninitialized constant Song (NameError)
```
I did a code search and didn't find reference on `auditing_enabled`. maybe you can educate me on this magic?

Reply:
Go to https://github.com/collectiveidea/audited and search for "https://github.com/collectiveidea/audited"